### PR TITLE
Update code intel roadmap

### PR DESCRIPTION
**Pull request by macraig**

### CODE REVIEW

**Comment by efritz** [comment]
@macraig No more goals?

**Comment by macraig** [comment]
@efritz I'm trying to keep the HB more high level and less task focused. Tasks will still be tracked in Productboard and Github :)

**Review by efritz — APPROVED** [review]

File: handbook/engineering/code-intelligence/goals.md [hunk 1]
```diff
@@ -22,120 +22,76 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
...
 
 **Milestones:**
 
-1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
-    - ✅ Automatic inference for projects that use the Java plugin
-    - Automatic inference for projects that use the Android plugin
-    - Publish Gradle plugin.
-    - 🔄 Update and improve documentation to ensure a good user experience.
-    
+1. ✅ Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
```
Comment (macraig) [inline handbook/engineering/code-intelligence/goals.md:54]:
```suggestion
1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
```
Work for Gradle support is complete but we still need to index repos on Cloud

File: handbook/engineering/code-intelligence/goals.md [hunk 2]
```diff
@@ -22,120 +22,76 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
...
+
     **Outcome:** 5 popular open source Gradle repositories have up-to-date precise code intelligence on Sourcegraph Cloud.
 
-1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
-    - ✅ Automatic inference for projects that use the Java compiler plugin
-    - Automatic inference for projects that use the Scala/Kotlin plugins
-    - Publish Maven plugin.
-    - 🔄 Update and improve documentation to ensure a good user experience.
-    
+1. ✅ Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
```
Comment (macraig) [inline handbook/engineering/code-intelligence/goals.md:58]:
```suggestion
1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Maven repositories.
```
Work for Maven support is complete but we still need to index repos on Cloud to mark milestone completion

**Pull request merged**